web/embed_temp.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
```

Comment:
Lets plan (in a follow-on PR) to split this file into three separate embeddings. Hopefully we can reduce code duplication by having some shared html and or js files.